includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -2,14 +2,238 @@
 
 class DesignSystemGlobalFooterModel extends WikiaModel {
 	const DEFAULT_LANG = 'en';
-
-	private $wikiId;
+	const PRODUCT_WIKIS = 'wikis';
+	const PRODUCT_FANDOMS = 'fandoms';
+
+	private $hrefs = [
+		'default' => [
+			'fan-communities' => 'http://fandom.wikia.com/explore',
+			'about' => 'http://www.wikia.com/about',
+			'careers' => 'https://careers.wikia.com',
+			'press' => 'http://fandom.wikia.com/press',
+			'contact' => 'http://fandom.wikia.com/contact',
+			'wikia-gives-back' => 'http://www.wikia.com/wikiagivesback',
+			'terms-of-use' => 'http://www.wikia.com/Terms_of_use',
+			'privacy-policy' => 'http://www.wikia.com/Privacy_Policy',
+			'global-sitemap' => 'http://www.wikia.com/Sitemap',
+			'local-sitemap' => '/wiki/Local_Sitemap',
+			'local-sitemap-fandom' => 'http://fandom.wikia.com/local-sitemap',
+			'api' => 'http://api.wikia.com/wiki/Wikia_API_Wiki',
+			'community-central' => 'http://community.wikia.com/wiki/Community_Central',
+			'support' => 'http://community.wikia.com/wiki/Special:Contact',
+			'create-new-wiki' => 'http://www.wikia.com/Special:CreateNewWiki',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "create-new-wiki" 12 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)